pytest_jira.py

```diff
@@ -18,12 +18,21 @@
 class JiraHooks(object):
     issue_re = r"([A-Z]+-[0-9]+)"
 
-    def __init__(self, url, username=None, password=None, verify=True):
+    def __init__(
```

Comment:
in order to decrease number of parameters, you can pass structures instead of values, like:

```python
class JiraSiteConnection(object):
    def __init__(self, url, user, password, verify):
         # ...
     def connect(self):
         # ...
     def is_connected(self):
         # ...
     def get_issue(self, issue):
         # ...

class JiraIssueResolver(object):
    def __init__(self, components, version):
         # ...
     def vote(self, issue):
         # ...

class JiraHooks(object):
    def __init__(self, connection, vote_policy):
         # ...
```